pyodide-build/pyodide_build/create_xbuildenv.py

```diff
@@ -47,8 +47,9 @@ def get_relative_path(pyodide_root: Path, flag: str) -> Path:
         return Path(get_build_flag(flag)).relative_to(pyodide_root)
 
     pythoninclude = get_relative_path(pyodide_root, "PYTHONINCLUDE")
-    wasm_lib_dir = get_relative_path(pyodide_root, "WASM_LIBRARY_DIR")
     sysconfig_dir = get_relative_path(pyodide_root, "SYSCONFIGDATA_DIR")
+    # TODO: remove libs from the xbuildenv
+    wasm_lib_dir = pyodide_root / "packages" / ".libs"
```

Comment:
In the future, I think we should remove library artifacts from `xbuildenv`. I think it's strange that some selected libraries are included here.

WDYT @hoodmane? Are these libraries in the xbuildenv used in numpy or scipy CI?

Reply:
Not sure. I know that some people are using these, people have requested that we put more things in. I thought they were more for building packages that used numpy/scipy at build time.

Reply:
> we put more things in

In the long run, We'll probably need to find a separate place to host the built libraries. Of course, this is the same issue as hosting packages.